frontend/app/dashboard/dashboard-taxonomy/dashboard-taxonomy.component.ts

```diff
@@ -115,7 +115,8 @@ export class DashboardTaxonomyComponent implements OnInit {
   @Input() taxonomies: any;
   @Input() yearsMinMax: any;
   public currentTaxLevel = "Règne";
-  public yearRange = [0, 2019];
+  public currentYear: number = new Date().getFullYear();
+  public yearRange = [0, this.currentYear];
```

Comment:
Il n'y a pas besoin de définir `currentYear` comme une variable `public`, ni comme une variable de classe comme elle n'est pas utilisé côté frontend.
```suggestion
  public yearRange = [0, new Date().getFullYear()];
```